projects/elife.yaml

```diff
@@ -548,6 +548,29 @@ recommendations:
                     - bus-subjects--{instance}
                     - bus-collections--{instance}
 
+personalised-covers:
+    domain: False
```

Comment:
`domain: False` only if no link has to be sent outside (e.g. in emails), or if those link are directly to S3 (bad), is that correct?